yarn-project/p2p/src/services/reqresp/reqresp.ts

```diff
@@ -538,7 +538,7 @@ export class ReqResp {
 
     // Timeout errors are punished with high tolerance, they can be due to a geogrpahically far away peer or an
     // overloaded peer
-    if (e instanceof IndividualReqRespTimeoutError) {
+    if (e instanceof IndividualReqRespTimeoutError || (e instanceof CodeError && e.code === 'ERR_TIMEOUT')) {
```

Comment:
Pointing this change out. It would get rid of these types of errors `Unexpected error sending request to peer: CodeError: The operation was aborted`